ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml

```diff
@@ -143,6 +143,18 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- always_run: false
+  as: e2e-aws-gatewayapi-conformance
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_GATES: '[GatewayAPI=true]'
+      FEATURE_SET: CustomNoUpgrade
```

Comment:
We will need to update this once https://github.com/openshift/api/pull/2081 merges (which we want to happen as soon as possible).

Reply:
OK, once https://github.com/openshift/api/pull/2081 merges we could just set `FEATURE_SET: TechPreviewNoUpgrade`.
And finally we will promote to GA in 4.19 right?

Reply:
Should we hold this PR until https://github.com/openshift/api/pull/2081 get merged ?

Reply:
Regarding a hold until the feature is merged as TechPreview:

Pro: we won't need another change within a month or so.  We will still need another change to GA, so one change instead of two would be more efficient.

Con: we lose a month or so of conformance tests against OSSM 3.0 GA.

I vote no, it would be better to have the tests against OSSM 3.0.  @Miciah do you agree?